### DISCUSSION TITLE
Unexpected token when export data

### DISCUSSION THREAD

**Discussion by RedBlob** [Budibase/budibase#10674] [Help] [answered]
Hello,

Using the "Export data" function, an error message appears referring to the JSON at position 1

In position 1 seems to be the primary key of the PostgreSQL table, which, however, has been working correctly for months without any problems ( it is an integer variable that is correctly operational ).

What could be the problem ?

![Immagine 2023-05-23 173122](https://github.com/Budibase/budibase/assets/70526244/4b5123fc-255e-4b66-aa31-642c3a8c7fb6)
![Immagine 2023-05-23 172927](https://github.com/Budibase/budibase/assets/70526244/152e41c9-ee46-4e39-8144-1ac5cf4813a1)

Thank You for help

**Comment by melohagan** [comment]
Hey @RedBlob 

Could you please share the schema of your SQL table (the table CREATE script).

**Comment by RedBlob** [comment]
Hi @melohagan 

**This is the SQL for primary Key sequence :**

CREATE SEQUENCE IF NOT EXISTS public.id_flux_seq
    INCREMENT 1
    START 1
    MINVALUE 1
    MAXVALUE 9223372036854775807
    CACHE 1;

ALTER SEQUENCE public.id_flux_seq
    OWNER TO postgres;

GRANT ALL ON SEQUENCE public.id_flux_seq TO redblob;



**And this is the SQL for table :**


CREATE TABLE IF NOT EXISTS public.rfid_flux
(
    id_flux bigint NOT NULL DEFAULT nextval('id_flux_seq'::regclass),   
    codice text COLLATE pg_catalog."default" NOT NULL DEFAULT 'x'::text,
    nome text COLLATE pg_catalog."default" NOT NULL DEFAULT 'x'::text,
    indirizzo text COLLATE pg_catalog."default" NOT NULL DEFAULT 'x'::text,    
    flag_sync_db boolean DEFAULT false,
    flag_sync_loc boolean DEFAULT false,
    CONSTRAINT local_flux_pkey PRIMARY KEY (id_flux)
)

TABLESPACE pg_default;

ALTER TABLE IF EXISTS public.rfid_flux
    OWNER to postgres;

GRANT UPDATE, INSERT, SELECT ON TABLE public.rfid_flux TO redblob ;


Thank You for help.

**Reply by RedBlob** [reply]
Hello @melohagan ,
Have you suggestion for this problem ?
We must start the testing on production and the export function is an important feature for our users.
Thank You.
Best Regards

**Reply by melohagan** [reply, marked as answer]
Hey @RedBlob I believe this is a bug, and I currently have a PR out: https://github.com/Budibase/budibase/pull/10721
